### PR TITLE
[MOSIP-31733] property migration bugs fixed for platform up…

### PR DESCRIPTION
**Pull request by Mohanraj209**
…grade.

### CODE REVIEW

**Review by gsasikumar — CHANGES_REQUESTED** [review]

File: deployment/v3/utils/prop_migrator/Overview.md [hunk 1]
```diff
@@ -4,28 +4,28 @@
 
 # Migrator script
 
-This script automates the process of comparing and updating configurations between an existing old configuration repository and a new based configuration repository. It helps ensure that the latest configurations are applied while preserving any customizations made in the old repository. 
+This script automates the process of comparing and updating configurations between an existing old configuration branch and a latest configuration branch. It helps ensure that the latest configurations are applied while preserving any customizations made in the old config branch.
```
[the remark this answers is not in the document]
Comment (Mohanraj209) [reply deployment/v3/utils/prop_migrator/Overview.md:7]: Migrator script compares the property files from one version of config to another version, while performing the upgrade of v2 version to v3 we knew that v2 environments were using 1.1.5.5 config branch and v3 environments were using 1.2.0.1 so we compared these config branches using migrator and listed out the differences. so here the old configuration branch will be "1.1.5.5" and latest configuration branch will be '1.2.0.1'.

File: deployment/v3/utils/prop_migrator/Overview.md [hunk 2]
```diff
@@ -4,28 +4,28 @@
...
+This script automates the process of comparing and updating configurations between an existing old configuration branch and a latest configuration branch. It helps ensure that the latest configurations are applied while preserving any customizations made in the old config branch. 
 
-The script compares property files between an old repository and a latest repository, identifies the differences, and generates CSV files to summarize the results, then automates the process of updating property files based on information provided in knowledge base CSV files. It allows you to manage and prioritize property values, handle different versions of property files, and generate a manual configuration file for further customization.
+The script compares property files between an old config branch and a latest config branch, identifies the differences, and generates CSV files to summarize the results, then automates the process of updating property files based on information provided in knowledge base CSV files. It allows you to manage and prioritize property values, handle different versions of property files, and generate a manual configuration file for further customization.
```
[the remark this answers is not in the document]
Comment (Mohanraj209) [reply deployment/v3/utils/prop_migrator/Overview.md:9]: Migrator script compares the property files from one version of config to another version, while performing the upgrade of v2 version to v3 we knew that v2 environments were using 1.1.5.5 config branch and v3 environments were using 1.2.0.1 so we compared these config branches using migrator and listed out the differences. so here the old configuration branch will be "1.1.5.5" and latest configuration branch will be '1.2.0.1'.

File: deployment/v3/mosip/conf-secrets/README.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 This module generates and install secrets required by config-server.
 
+Note: The conf-secrets must not be deleted in any case of re-deployment, upgrade or migration process as we have separated the conf-secrets from config-server to ensure conf-secrets are not deleted.
```
[the remark this answers is not in the document]
Comment (Mohanraj209) [reply deployment/v3/mosip/conf-secrets/README.md:5]: @gsasikumar yes it is possible to take backup of conf-secrets and restore, this is just a precautionary message to the users to not delete conf-secrets without any backup.

[the remark this answers is not in the document]
Comment (gsasikumar) [reply deployment/v3/mosip/conf-secrets/README.md:5]: Can you raise a task for back up so we can consider this as resolved?

File: deployment/v3/utils/prop_migrator/Overview.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gsasikumar) [reply deployment/v3/utils/prop_migrator/Overview.md:7]: Can we make this sentence better to reflect the above clearly?

File: deployment/v3/mosip/conf-secrets/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Mohanraj209) [reply deployment/v3/mosip/conf-secrets/README.md:5]: sure @gsasikumar

[the remark this answers is not in the document]
Comment (Mohanraj209) [reply deployment/v3/mosip/conf-secrets/README.md:5]: created task for the above https://mosip.atlassian.net/browse/MOSIP-32082

**Review by gsasikumar — APPROVED** [review]

**Pull request merged**